### PR TITLE
Make folder for Office Hours notes

### LINKED ISSUE #12: SourceCred office hours: 2019-01-10
SourceCred hosted its first office hours on Thursday, Jan 10 2019.

Here's the recording: https://www.youtube.com/watch?v=3NgANZNdSlo&feature=youtu.be&t=1

Thank you to @BrianLitwin for the [notes](https://github.com/sourcecred/mission/issues/12#issuecomment-453997944)

Attendees: @decentralion, @wchargin, @anthrocypher, @BrianLitwin, @lpatmo, @sternhenri, @dignifiedquire, (pls ping me to be added if I missed you)

### LINKED ISSUE #13: SourceCred Office Hours (2) - 2019-01-24
We're planning to have another round of SourceCred office hours on Thursday, Jan 24 2019, at 12PM PT / 3PM ET.

Here's the Zoom link for the meeting: https://protocol.zoom.us/j/702559487

### LINKED ISSUE #18: Put Office Hours notes/video into dedicated folder
Again, @wpank provided some good [examples] of repos that are doing this effectively.  

[examples]: https://github.com/sourcecred/mission/issues/17#issuecomment-464112696

### PR DESCRIPTION
**Pull request by BrianLitwin**
closes #12
closes #13
closes #18

Test plan:
make sure notes are in files, dates are correct, links are correct,
lists of attendees are representative

### CODE REVIEW

File: office-hours-notes/2019-01-10.md [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+* [SourceCred/SourceCred](https://github.com/sourcecred/sourcecred) will contain the codebase for the project
+* The most exciting recent development 🎉 is that we're starting to see new contributors/contributions to the project such as [@anthrocypher](https://github.com/anthrocypher) at [#1044](https://github.com/sourcecred/sourcecred/pull/1044), [@brianlitwin](https://github.com/brianlitwin) at [#1032](https://github.com/sourcecred/sourcecred/pull/1032) and [@expravit](https://github.com/expravit) at [#1031](https://github.com/sourcecred/sourcecred/pull/1031).
+* A big 2019 Q1 priority will be community building for the project. SourceCred is really a **social technology** in the sense that we're trying to come to a social consensus about who deserves cred and which things we value in a project. Building a community of people who are excited to try SourceCred is vital to the success of the project.
+* Areas of focus to promote community-building: documentation, videos and events such as this Office Hours.
+
+####  How do we use gitHub's GraphQl service? [2:03](https://www.youtube.com/watch?v=3NgANZNdSlo#t=2m03s)
```
Comment (teamdandelion) [inline office-hours-notes/2019-01-10.md]: Nit: s/gitHub/GitHub/

File: office-hours-notes/2019-01-10.md [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+
+* The first hurdle is that GitHub's GraphQL service is paginated, so you can only request 100 nodes at a time, eg 100 Issues or Comments at a time. So we request nodes in batches one after the other if we want more than 100.
+* We also want to request multiple kinds of data in one query because there is a recursive structure to the data: A Reposiotry will have a certain number of Pull Requests, each of which will have a certain number of Pull Request Reviews, etc.
```
Comment (teamdandelion) [inline office-hours-notes/2019-01-10.md]: nit: s/Reposiotry/Repository/

**Review by teamdandelion — APPROVED** [review]
Thank you!

**Comment by BrianLitwin** [comment]
Thanks for the review @decentralion! Updated according to your notes.

**Pull request merged**